### PR TITLE
Change checkpoint urls and add op/base chains for each commit sync tests

### PR DESCRIPTION
**Pull request by kamilchodola**
Change checkpoint urls to fix sepolia and mainnet sync
Add op/base sepolia/mainnet for each commit sync. 

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [X] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [X] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by MarekM25 — APPROVED** [review]

**Review by benaadams — APPROVED** [review]

File: .github/workflows/sync-supported-chains.yml
Comment (kamilchodola) [reply .github/workflows/sync-supported-chains.yml:117]: not at all... didn't reached the stage of analyzing sync logs yet so that's why I didn't noticed that :D thanks!

File: scripts/workflow_config/sync_testnets_matrix.json
Comment (brbrr) [inline scripts/workflow_config/sync_testnets_matrix.json:64]: I wonder if we really need such a big VM here.

File: .github/workflows/sync-supported-chains.yml
Comment (kamilchodola) [reply .github/workflows/sync-supported-chains.yml:56]: So it is kind of a hack ;)
At first I kept it as it was - suddenly I exceeded max length of Linode (32 chars)
Then changed it to ${{ matrix.config.network }}-${{ github.run_id }} - had a collision on filtering that base-sepolia-id and sepolia-id where treated as duplicates (I need to always find exactly one machine to remove it)
Wanted to switch order of params to fix above - Lindoe machine name needs to start with a letter...
So added the t- prefix meaning "tests" :DDD 

Hope you will understand my struggle

File: scripts/workflow_config/sync_testnets_matrix.json
Comment (kamilchodola) [reply scripts/workflow_config/sync_testnets_matrix.json:64]: As still on draft - testing what machines shall I use - will add debug "du -h" at the end to see exactly what types of machines shall I use.

Also will test out if I can use NonValidatorNode for op stack - if works will be definitelly smaller

File: .github/workflows/sync-supported-chains.yml
Comment (kamilchodola) [reply .github/workflows/sync-supported-chains.yml]: cleaned

File: .github/workflows/sync-supported-chains.yml
Comment (brbrr) [reply .github/workflows/sync-supported-chains.yml:56]: totally. that's supper annoying to deal with. same issue with smoke tests workflows 😬

**Review by brbrr — APPROVED** [review]

**Comment by kamilchodola** [comment]
Will merge once OP fixes will land to not cause failed runs

**Pull request merged**
